README.md

```diff
@@ -1,13 +1,17 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was to build a survey with form elements in React, using state and compnents.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started by making the questions in one file. I used input text, radiobuttons, select options and checkboxes.
+Then I split the questions into separate components, which are mounted from the form component. To start with I printed alla questions on one 
+page and the showed a summer of the answers when clicking the submit button. After the I changed the survey so that it only showed one question 
+at a time. I added a next-button and a back-button, so the user can bothe go forward and back. When all the question are answered a summary is shown.
+
+The way of building this project in small steps helped me understand components and state, how the different components fit together and where the
```

Comment:
Really beautiful page Sara, you should be proud of yourself with what you have accomplished with such a short time learning about react 😍 🌟